extensions/blocks/google-calendar/google-calendar.php

```diff
@@ -35,31 +35,30 @@ function register_block() {
  * @return string
  */
 function load_assets( $attr ) {
-	$width   = isset( $attr['width'] ) ? $attr['width'] : '800';
 	$height  = isset( $attr['height'] ) ? $attr['height'] : '600';
 	$url     = isset( $attr['url'] )
 		? \Jetpack_Gutenberg::validate_block_embed_url( $attr['url'], array( 'calendar.google.com' ) ) :
 		'';
 	$classes = \Jetpack_Gutenberg::block_classes( 'google-calendar', $attr );
 
+	\Jetpack_Gutenberg::load_assets_as_required( 'google-calendar' );
+
 	if ( empty( $url ) ) {
 		return;
 	}
 
 	if ( class_exists( 'Jetpack_AMP_Support' ) && \Jetpack_AMP_Support::is_amp_request() ) {
 		return sprintf(
-			'<div class="%1$s"><amp-iframe src="%2$s" frameborder="0" style="border:0" scrolling="no" width="%3$d" height="%4$d" sandbox="allow-scripts allow-same-origin" layout="responsive"></amp-iframe></div>',
+			'<div class="%1$s"><amp-iframe src="%2$s" frameborder="0" style="border:0" scrolling="no" height="%3$d" sandbox="allow-scripts allow-same-origin" layout="responsive"></amp-iframe></div>',
```

Comment:
This change appears to have broken Google Calendar on AMP pages, since there is no `width` defined, and the `layout` was not changed from `responsive` to `fixed-height`. Opened PR to fix in #17251.